### ISSUE TITLE
What are the trends for optimal training results for training losses and validation losses

### ISSUE THREAD

**Issue by gittzb** [MIC-DKFZ/nnUNet#2598]
When training my own focal data set (binary classification) set 1000 epochs, after the training, my validation loss has converged, but the training loss has been declining, is this the best state?

**Comment by gittzb** [comment]
![progress](https://github.com/user-attachments/assets/4eba2218-3004-48c7-a957-8d98b32ca024)

**Comment by gittzb** [comment]
![progress](https://github.com/user-attachments/assets/26a5e3ee-0c73-47b7-b6b6-e1804c954f21)

**Comment by Karol-G** [comment]
Hey,

Is this issue still relevant? Otherwise, I will close it in 14 days.

Best,
Karol

**Comment by Karol-G** [comment]
Closing this issue due to inactivity.
